src/sparsezoo/analysis/helpers.py

```diff
@@ -0,0 +1,258 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Tuple
+
+import numpy as np
+from onnx import ModelProto, NodeProto, numpy_helper
+
+
+__all__ = [
+    "get_layer_and_op_counts",
+    "get_node_four_block_sparsity",
+    "get_node_four_block_sparsity_sizes",
+    "get_node_sparsity",
+    "get_node_sparsity_sizes",
+    "get_zero_point",
+    "is_four_block_sparse_layer",
+    "is_parameterized_prunable_layer",
+    "is_quantized_layer",
+    "is_sparse_layer",
+]
+
+
+def get_initializer(model: ModelProto, initializer_name: str) -> np.ndarray:
+    """
+    Helper function to find initializers by name in model graph
+
+    :param model: model used to find initializer
+    :param initializer_name: name of initializer being returned
+    :return: initalizer if found, None otherwise
+    """
+    for initializer in model.graph.initializer:
+        if initializer.name == initializer_name:
+            return initializer
+
+    return None
+
+
+def get_node_sparsity_sizes(model: ModelProto, node: NodeProto) -> Tuple[int, int]:
+    """
+    :param model: model in which node's parameter exists
```

Comment:
nit: our docs compiler has trouble with apostrophes since it can look like an unclosed string - should reword to avoid apostrophe - it's an art :)

Reply:
there's quite a few of these - will just put this one, but please check for all